File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


while True:
    height = get_int("Height: ")
    if 0 < height <= 8:
        break


for i in range(height):
    for j in range(height):
        if (i + j < height - 1):
            print(" ", end="")
        else:
            print("#", end="")
    print()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    //continue to ask this question until user cooperates
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //for each row
    for (int i = 0; i < n; i++)
    {

        //s accounts for how many spaces to print
        //spaces = height - row - 1
        //s < n - i - 1, you don't want to go thru the 8th iteration
        //8th iteration will print 9 hases
        for (int s = 0; s < n - i - 1; s++)
        {
            //print a space
            printf(" ");
        }

        //for each column
        //you want row number = hash number
        //row index starts at 0
        for (int j = 0; j <= i; j++)
        {
            //print a brick
            printf("#");
        }

        //start the next row
        printf("\n");

    }
}